dependencies/che-devfile-registry/build/scripts/index.sh

```diff
@@ -19,6 +19,7 @@ for meta in "${metas[@]}"; do
       cat <<< "$(yq -y --arg metadir "${META_DIR}" '.links.devWorkspaces |= . +
       {"eclipse/che-theia/latest": "/\($metadir)/devworkspace-che-theia-latest.yaml",
       "eclipse/che-idea/latest": "/\($metadir)/devworkspace-che-idea-latest.yaml",
+      "che-incubator/che-code/insiders": "/\($metadir)/devworkspace-che-code-insiders.yaml",
       "che-incubator/che-code/latest": "/\($metadir)/devworkspace-che-code-latest.yaml",}' "${meta}")" > "${meta}"
```

Comment:
there's an extra comma in here (?)...

```suggestion
      "che-incubator/che-code/latest": "/\($metadir)/devworkspace-che-code-latest.yaml"}' "${meta}")" > "${meta}"
```